src/main/java/Entities/Board.java

```diff
@@ -22,10 +22,27 @@ public Board(Player[] players, Cell[] cells, Object[][] propertyData){
             String name = (String) data[0];
             String colour = (String) data[1];
             int cost = (int) data[2];
-            int rent = (int) data[3];
-            int mortgage = (int) data[4];
-            int houseCost = (int) data[5];
-            properties[i] = new Property(name, colour, cost, rent, mortgage, houseCost);
+            int houseCost = (int) data[3];
+            int rent = (int) data[4];
+            int rent1H = (int) data[5];
+            int rent2H = (int) data[6];
+            int rent3H = (int) data[7];
+            int rent4H = (int) data[8];
+            int rentHotel = (int) data[9];
+            Player owner = null;
+            if ((int) data[10] != 0) {
+                owner = players[(int) data[10]];
```

Comment:
I denote the Player which owns the property by their position in the Player[] array of all Players (indexed starting at 1). Default value for no owner is 0.